lib/Dialect/Struct/IR/Ops.cpp

```diff
@@ -134,9 +140,13 @@ LogicalResult checkSelfType(
 //===------------------------------------------------------------------===//
 namespace {
 
-inline LogicalResult msgOneFunction(EmitErrorFn emitError, const Twine &name) {
-  return emitError() << "must define exactly one '" << name << "' function";
-}
+// inline LogicalResult msgOneFunction(EmitErrorFn emitError, const Twine &name) {
+//   return emitError() << "must define exactly one '" << name << "' function";
+// }
+
+// inline LogicalResult msgDuplicateFunction(EmitErrorFn emitError, const Twine &name) {
+//   return emitError() << "must not define multiple '" << name << "' functions";
+// }
```

Comment:
If we don't need these we should remove them before merging

Reply:
@raghav198 These are the ones I mentioned in the meeting that cause the tests to fail is CI